cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,55 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 2.6.3 <http://docs.cask.co/cdap/2.6.3/index.html>`__
+=============================================================
+
+CDAP Bug Fixes
+--------------
+
+- Replace use of CDAP transaction snapshot codecs with the Tephra versions instead
+  (`CDAP-2496 <https://issues.cask.co/browse/CDAP-2496>`__)
+  
+  
+Upgrade Instructions
```

Comment:
We don't need to specify any additional information here. Installation and Configuration section should already cover it.